site/docs/hosting/heroku.md

```diff
@@ -135,7 +135,7 @@ Let's take a look at our code above:
   For creating [Environment Variables in Heroku](https://www.freecodecamp.org/news/using-environment-variables-the-right-way/), head over to [this guide](https://devcenter.heroku.com/articles/config-vars).
 - `secretPath`: It could be our `BOT_TOKEN` or any random string. It is best practice to hide our bot path as [explained by Telegram](https://core.telegram.org/bots/api#setwebhook).
 
-::: tip ⚡ Optimization (optional)
+::: tip ⚡ Optimization (Optional)
```

Comment:
For consistency with heading style, and as per https://github.com/grammyjs/website/pull/30#discussion_r735909521
```suggestion
::: tip ⚡ Optimization (optional)
```